x-pack/plugins/ingest_manager/server/routes/agent/index.ts

```diff
@@ -111,6 +113,20 @@ export const registerRoutes = (router: IRouter) => {
     })
   );
 
+  // Agent actions
+  router.post(
+    {
+      path: AGENT_API_ROUTES.ACTIONS_PATTERN,
+      validate: PostNewAgentActionRequestSchema,
+      options: { tags: [`access:${PLUGIN_ID}-all`] },
+    },
+    postNewAgentActionHandlerBuilder({
+      getAgent: AgentService.getAgent,
+      getSavedObjectsClientContract: getInternalUserSOClient,
```

Comment:
Actually we want to use the SOClient that comme from the request directly 
like this here `const soClient = context.core.savedObjects.client;` and not the internal user one

Reply:
okay will make the change.